### ISSUE TITLE
Alexnet 코드 리뷰

### ISSUE THREAD

**Issue by deblee2** [hoiho77/CNN-model-implementation#2]
## alexnet.py

코드 잘 봤습니다. yskim0님 말대로 가독성이 좋네요! 흥미로운 점은 각 레이어마다 BatchNormalization을 추가해주신 점인데요. 제가 알기론 Batch Normalization이 학습을 더 향상시키는 걸로 알고있는데 유무의 차이가 어느정도인지 궁금합니다! 그리고 4, 5번째 conv layer에서 kernel_size값이 논문에서는 (3,3)으로 나와있는데 (1,1)로 설정하신 이유가 궁금합니다!
